lib/oppen.rb

```diff
@@ -94,72 +102,74 @@ def trim_trailing_whitespaces? = @trim_trailing_whitespaces
 
     def upsize_stack? = @upsize_stack
 
-    # Default config for Oppen usage
+    # Default config for Oppen usage.
+    #
     # @return [Config]
     def self.oppen
       new
     end
 
-    # Default config for Wadler usage
+    # Default config for Wadler usage.
+    #
     # @return [Config]
     def self.wadler(eager_print: true, trim_trailing_whitespaces: true, upsize_stack: true)
-      new(indent_anchor: IndentAnchor::ON_BEGIN, eager_print:, trim_trailing_whitespaces:, upsize_stack:)
+      new(eager_print:, indent_anchor: IndentAnchor::ON_BEGIN, trim_trailing_whitespaces:, upsize_stack:)
     end
   end
 
   # @param value [String]
-  # @param width [Integer] token width that defaults to value.length
+  # @param width [Integer] token width that defaults to value.length.
   #
-  # @return [Oppen::Token::String] a new String token
+  # @return [Token::String] a new String token.
   def self.string(value, width: value.length)
     Token::String.new(value, width:)
   end
 
   # @see Token::Whitespace
   #
-  # @return [Oppen::Token::Whitespace] a new Whitespace token.
+  # @return [Token::Whitespace] a new Whitespace token.
   def self.whitespace(value)
     Token::Whitespace.new(value, width: value.bytesize)
   end
 
-  # @param str [String]
-  # @param line_continuation [String] If a new line is needed display this string before the new line
-  # @param offset [Integer]
-  # @param width [Integer] token width that defaults to str.length
+  # @param str               [String]
+  # @param line_continuation [String]  If a new line is needed display this string before the new line.
+  # @param offset            [Integer]
+  # @param width             [Integer] Token width that defaults to str.length.
   #
-  # @return [Oppen::Token::Break] a new Break token
+  # @return [Token::Break] a new Break token.
   def self.break(str = ' ', line_continuation: '', offset: 0, width: str.length)
     Token::Break.new(str, width:, line_continuation:, offset:)
   end
 
-  # @param line_continuation [String] If a new line is needed display this string before the new line
-  # @param offset [Integer]
+  # @param line_continuation [String] If a new line is needed display this string before the new line.
```

Comment:
The issue with before and after, it's that these are relative terms, and can be ambiguous.

I propose:

```suggestion
  # @param line_continuation [String] If a new line is needed display this string at the beginning of the new line, at the specified anchor position.
```

Or something like that, with hyperlinks to anchors too, and also `@see` for them. What's important here is that I'm trying to eliminate ambiguity and room for interpretation.

The best doc is the one that is intimately interlinked making effectively a wiki.